src/lib_json/json_value.cpp

```diff
@@ -190,6 +191,7 @@ static inline void releaseStringValue(char* value, unsigned) {
 
 namespace Json {
 
+#if JSON_USE_EXCEPTION
```

Comment:
That code contains 'throw'. When exceptions are disabled in the compiler, throw is unavailable.